src/main/java/am/ik/rsocket/InteractionModel.java

```diff
@@ -113,4 +142,16 @@ static Mono<String> scanToMono(Scanner scanner) {
 				.collectList() //
 				.map(list -> String.join(System.lineSeparator(), list));
 	}
+
+	static Mono<Void> removeOutputFileMono(Args args) {
+		return Mono.fromRunnable(() -> args.outputFile()
+				.map(path -> {
+					try {
+						Files.deleteIfExists(path);
```

Comment:
Including this operation in a Mono chain looks weird.
This can be done earlier.

Reply:
Same as above. I was trying to avoid changing a lot of code.